test/test_functional_tensor.py

```diff
@@ -692,6 +433,250 @@ def test_scale_channel(self):
         assert_equal(scaled_cpu, scaled_cuda.to('cpu'))
 
 
+class TestAffine:
+
+    @pytest.mark.parametrize('device', cpu_and_gpu())
+    @pytest.mark.parametrize('height, width', [(26, 26), (32, 26)])
+    @pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
+    def test_identity_map(self, device, height, width, dt):
+        # Tests on square and rectangular images
+        scripted_affine = torch.jit.script(F.affine)
+        tensor, pil_img = _create_data(height, width, device=device)
+
+        if dt == torch.float16 and device == "cpu":
+            # skip float16 on CPU case
+            return
+
+        if dt is not None:
+            tensor = tensor.to(dtype=dt)
+
+        # 1) identity map
+        out_tensor = F.affine(tensor, angle=0, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], interpolation=NEAREST)
+
+        assert_equal(tensor, out_tensor, msg="{} vs {}".format(out_tensor[0, :5, :5], tensor[0, :5, :5]))
+        out_tensor = scripted_affine(
+            tensor, angle=0, translate=[0, 0], scale=1.0, shear=[0.0, 0.0], interpolation=NEAREST
+        )
+        assert_equal(tensor, out_tensor, msg="{} vs {}".format(out_tensor[0, :5, :5], tensor[0, :5, :5]))
+
+    @pytest.mark.parametrize('device', cpu_and_gpu())
+    @pytest.mark.parametrize('height, width', [(26, 26)])
+    @pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
```

Comment:
as we always parametrize over the same dtypes, perhaps we could define this list as a class variable, like `ALL_DTYPES = (None, torch.float32, torch.float64, torch.float16)`?

Reply:
Okay sure.